### ISSUE TITLE
Reference to old argument name of `torch.var` in `LayerNorm` documentation page

### ISSUE THREAD

**Issue by Ecpii** [pytorch/pytorch#166804]
There is the following issue on this page: https://docs.pytorch.org/docs/stable/generated/torch.nn.LayerNorm.html

> The variance is calculated via the biased estimator, equivalent to torch.var(input, unbiased=False).

The `unbiased` argument to `torch.var` should instead be `correction=0`.

**Comment by mohsinm-dev** [comment]
I think it is correct and not deprecated.

**Comment by Ecpii** [comment]
Correct me if I'm wrong (just starting to use PyTorch), but I see this note on the docs of [`torch.var`](https://docs.pytorch.org/docs/stable/generated/torch.var) (with respect to the `correction` argument):

> Changed in version 2.0: Previously this argument was called unbiased and was a boolean with True corresponding to correction=1 and False being correction=0.

**Comment by yashwantbezawada** [comment]
Hi! I'd like to work on this issue.

The fix is straightforward: update the LayerNorm documentation to use the current `correction=0` argument instead of the deprecated `unbiased=False` when referencing `torch.var`.

As confirmed in the [`torch.var` docs](https://docs.pytorch.org/docs/stable/generated/torch.var.html):
> Changed in version 2.0: Previously this argument was called unbiased and was a boolean with True corresponding to correction=1 and False being correction=0.

I'll update the docstring in `torch/nn/modules/normalization.py` and submit a PR shortly if there are no objections.

Thanks!
